dbt/adapters/databricks/impl.py

```diff
@@ -817,4 +817,5 @@ def _describe_relation(
 
         if not relation.is_hive_metastore():
             results["information_schema.tags"] = adapter.execute_macro("fetch_tags", kwargs=kwargs)
+        results["show_tblproperties"] = adapter.execute_macro("fetch_tbl_properties", kwargs=kwargs)
```

Comment:
Core idea with MV/ST and now incremental is to check for what's in Databricks and compare against what is configured in dbt project.  This is grabbing the tblproperties from the table that has already been deployed to check if they have changed.